packages/xo-web/src/common/select-objects.js

```diff
@@ -790,13 +790,16 @@ export class SelectResourceSetsSr extends React.PureComponent {
   set value (value) {
     this.refs.select.value = value
   }
-  _getSrs = createSelector(
-    () => this.props.resourceSet,
-    ({ objectsByType }) => {
-      const { predicate } = this.props
-      const srs = objectsByType['SR']
-      return sortBy(predicate ? filter(srs, predicate) : srs, 'name_label')
-    }
+
+  _getSrs = createSort(
+    createFilter(
+      () => this.props.resourceSet.objectsByType.SR,
+      createSelector(
```

Comment:
No need selector.

Reply:
This one may be useless indeed. I think that's my bad :angel:

Reply:
it's changed when the template is changed